### DISCUSSION TITLE
Using PEST in flopy

### DISCUSSION THREAD

**Discussion by Fuadqr** [modflowpy/flopy#1297] [Q&A]
Is there any tutorial/notebooks showing me how to use the PEST in my model to estimate the hydraulic parameters (let's say the horizontal hydraulic conductivity) using some observed head data?

Ps. I've checked the notebooks regarding PEST utilities and how to write the template file (https://github.com/modflowpy/flopy/blob/develop/examples/Notebooks/flopy3_PEST.ipynb)
but other than that I couldn't find how to use it in order to estimate the best parameter using some observation data

**Comment by christianlangevin** [comment]
You probably want to use pyemu.  There is an example for MODFLOW 6 here: https://github.com/pypest/pyemu/blob/develop/examples/pstfrom_mf6.ipynb
